pkg/m3admin/namespace.go

```diff
@@ -0,0 +1,115 @@
+package m3admin
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/gogo/protobuf/jsonpb"
+	namespace "github.com/m3db/m3/src/dbnode/generated/proto/namespace"
+	"github.com/m3db/m3/src/query/generated/proto/admin"
+	myspec "github.com/m3db/m3db-operator/pkg/apis/m3dboperator/v1"
+)
+
+const (
+	// TODO(PS) add to spec for crd?
+	// defaults for namespace creation
+	_namespaceURI                                                    = "/api/v1/namespace"
+	_defaultName                                                     = "default"
+	_defaultBootstrapEnabled                                         = true
+	_defaultFlushEnabled                                             = true
+	_defaultWritesToCommitLog                                        = true
+	_defaultCleanupEnabled                                           = true
+	_defaultSnapshotEnabled                                          = false
+	_defaultRepairEnabled                                            = false
+	_defaultRententionOptionRetentionPeriodNanos                     = int64(172800000000000)
+	_defaultRententionOptionBlockSizeNanos                           = int64(7200000000000)
+	_defaultRententionOptionBufferFutureNanos                        = int64(600000000000)
+	_defaultRententionOptionBufferPastNanos                          = int64(600000000000)
+	_defaultRententionOptionBlockDataExpiry                          = true
+	_defaultRententionOptionBlockDataExpiryAfterNotAccessPeriodNanos = int64(300000000000)
+	_defaultIndexOptionenabled                                       = true
+	_defaultIndexOptionBlockSizeNanos                                = int64(7200000000000)
+)
+
+//NamespaceCreate will create a namespace
```

Comment:
nit: comments should have space between // and first char.

i.e. `// Namespace...`